packages/mergebot/src/run.ts

```diff
@@ -26,6 +26,7 @@ const args = yargs(process.argv.slice(2))
     "show-extended": { alias: ["s3"], type: "boolean", desc: "display extended info" },
     "show-actions": { alias: ["s4"], type: "boolean", desc: "display actions" },
     "show-mutations": { alias: ["s5"], type: "boolean", desc: "display mutations" },
+    "project-only": { type: "boolean", desc: "Only change the project board, nothing else." },
```

Comment:
project-only is useful for local runs but not much else, so I don't know if it's worth keeping. I left it in because maybe it'll be useful elsewhere.